build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'net.sf.kxml'
+version = '2.3.1'
+
+sourceCompatibility = 1.7
```

Comment:
Should be lower for CLDC compat.

Reply:
Sorry, I don't know much (anything :-) about CLDC.
Please feel free to change the version to whatever is needed.

What I was able to find was this:
  "CLDC 8 is a subset of Java SE 8 that incorporates the language, API, and VM updates."
  (http://docs.oracle.com/javame/config/cldc/opt-pkgs/api/cldc/api/index.html)

If there is a way to check compatibility automatically, it would be best :-)

Reply:
Wasn't aware there is a new version... I think we should keep compatibility with CLDC 1.1 until a major version bump. On wikipedia, I found:  "... all Java ME platforms are currently restricted to JRE 1.3 features and use that version of the class file format (internally known as version 47.0)".